packages/editor/src/components/copy-handler/index.js

```diff
@@ -46,12 +46,12 @@ class CopyHandler extends Component {
 	}
 
 	onCut( event ) {
-		const { multiSelectedBlockClientIds } = this.props;
+		const { selectedBlockClientIds } = this.props;
 
 		this.onCopy( event );
 
-		if ( multiSelectedBlockClientIds.length ) {
-			this.props.onRemove( multiSelectedBlockClientIds );
+		if ( selectedBlockClientIds && selectedBlockClientIds.length ) {
```

Comment:
```js
const selectedBlockClientIds = selectedBlockClientId ? [ selectedBlockClientId ] : getMultiSelectedBlockClientIds();
```
Should there be a check whethere there is more than one id in here?

Reply:
You're right 👍

Reply:
Actually, I'd add a boolean flag `hasMultiSelection` it's more secure